### PR TITLE
Modify responses with userId by replacing userInfo 

### PR DESCRIPTION
**Pull request by madushajg**
## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: components/api/core/api_impl.bal [hunk 1]
```diff
@@ -133,32 +136,29 @@ public function createOrg(http:Request createOrgReq, gen:OrgCreateRequest create
...
-        return buildUnknownErrorResponse();
-    } else {
-        log:printDebug(io:sprintf("New organization \'%s\' added to REGISTRY_ORG_USER_MAPPING. Author : %s", orgName, userId));
-        return buildSuccessResponse();
-    }
-}
-
 public function getOrg(http:Request getOrgReq, string orgName) returns http:Response {
     json | error res = db:getOrganization(orgName);
+    string ERROR_MESSAGE = "Unable to fetch organization";
```
Comment (nadundesilva) [inline components/api/core/api_impl.bal]: Let's make this "const ERROR_MESSAGE". There must be optimizations if it is a const

File: components/api/constants/constants.bal [hunk 2]
```diff
@@ -42,3 +42,4 @@ public const string COOKIE_HEADER = "Cookie";
 public const string TOKEN_COOKIE_KEY = "chpat";
 
 public const string ROLE_ADMIN = "admin";
+public const string DEFAULT_IMAGE_VISIBILITY = "PUBLIC";
```
Comment (nadundesilva) [inline components/api/constants/constants.bal]: New line at the end of file

File: components/api/core/utils.bal [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+        log:printError(io:sprintf("Unexpected error occured while inserting org-user mapping. user : %s, Organization : %s", userId, orgName),
+                                err = orgUserRes);
+        return buildUnknownErrorResponse();
+    } else {
+        log:printDebug(io:sprintf("New organization \'%s\' added to REGISTRY_ORG_USER_MAPPING. Author : %s", orgName, userId));
+        return buildSuccessResponse();
+    }
+}
+
+function buildResponseWithUserInfo (json payload, string userId, string index) returns json | error {
```
Comment (nadundesilva) [inline components/api/core/utils.bal]: Let's update "index" to "field". Also can't the method just get userId from the payload[index] ? Do we need to pass the userId ?

File: components/api/core/api_impl.bal (hunk 1 above)
Comment (madushajg) [reply components/api/core/api_impl.bal]: I think we can't make it const since it defines inside a function.

**Pull request merged**